### PR TITLE
Set screenshot output folder

### PR DESCRIPTION
**Pull request by robert00s**
Close: #4230 

I'm not sure that `Screenshot directory` is located in right place in menu (Settings -> Screen)

### CODE REVIEW

**Comment by NiLuJe** [comment]
IIRC, we have a dedicated screensaver setting submenu ;).

File: frontend/apps/reader/modules/readermenu.lua [hunk 1]
```diff
@@ -178,6 +178,13 @@ function ReaderMenu:setUpdateItemTable()
         table.remove(self.menu_items.screensaver.sub_item_table, 9)
         table.insert(self.menu_items.screensaver.sub_item_table, ss_book_settings)
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readermenu.lua]: Shouldn't it go over here? :-)

Comment (Frenzie) [reply frontend/apps/reader/modules/readermenu.lua]: Oh sorry, misunderstood what it was.

File: frontend/ui/widget/screenshoter.lua [hunk 2]
```diff
@@ -54,6 +56,41 @@ function Screenshoter:onScreenshot(filename)
...
+    table.insert(buttons, {
+        {
+            text = _("Choose screenshot directory"),
+            callback = function()
+                UIManager:close(self.choose_dialog)
+                require("ui/downloadmgr"):new{
+                    onConfirm = function(path)
+                        G_reader_settings:saveSetting("screenshot_dir", path .. "/")
+                        UIManager:show(InfoMessage:new{
+                            text = T(_("Screenshoot directory set to:\n%1"), path),
```
Comment (Frenzie) [inline frontend/ui/widget/screenshoter.lua]:
```suggestion
                            text = T(_("Screenshot directory set to:\n%1"), path),
```

**Comment by poire-z** [comment]
Should the little identical stuff you put in readermenu and filemanagermenu be put with all the other common menu stuff in frontend/ui/elements/common_settings_menu_table.lua (or another included file)?

File: frontend/ui/widget/screenshoter.lua [hunk 3]
```diff
@@ -54,6 +56,41 @@ function Screenshoter:onScreenshot(filename)
...
+        {
+            text = _("Close"),
+            callback = function()
+                UIManager:close(self.choose_dialog)
+            end,
+        }
+    })
+    local screenshot_dir = G_reader_settings:readSetting("screenshot_dir") or DataStorage:getDataDir() .. "/screenshots/"
+    self.choose_dialog = ButtonDialogTitle:new{
+        title = T(_("Current screenshot directory:\n %1"), screenshot_dir),
```
Comment (Frenzie) [inline frontend/ui/widget/screenshoter.lua]: Minor final nitpick, but is that space intentional?

File: frontend/ui/elements/filemanager_menu_order.lua [hunk 4]
```diff
@@ -67,6 +67,7 @@ local order = {
     },
     screen = {
         "screensaver",
+        "screenshot",
```
Comment (Frenzie) [inline frontend/ui/elements/filemanager_menu_order.lua]: I think the Device menu might make more sense than the Screen menu, but I'm not really sure.

Comment (poire-z) [reply frontend/ui/elements/filemanager_menu_order.lua]: No strong opinion on that.
But we might visit the `Screen>` menu more often than the `Device>` one, so, as something of little regular use, I'd put it in `Device>` with the other misc things.

File: frontend/ui/widget/screenshoter.lua (hunk 3 above)
Comment (robert00s) [reply frontend/ui/widget/screenshoter.lua]: This is parametr of `ButtonDialogTitle` so `title=` should be indent.

File: frontend/ui/elements/filemanager_menu_order.lua (hunk 4 above)
Comment (robert00s) [reply frontend/ui/elements/filemanager_menu_order.lua]: I also thought about put it in `Device>`.

File: frontend/ui/widget/screenshoter.lua (hunk 3 above)
Comment (poire-z) [reply frontend/ui/widget/screenshoter.lua]: I think he means the one between `\n %1` (which adds some small indent on the 2nd line with the directory path) - but we have it in quite a few places, so either intentional, or it replicated thru cut & paste :)

Comment (robert00s) [reply frontend/ui/widget/screenshoter.lua]: :)

**Pull request merged**